Source/Scene/Model.js

```diff
@@ -2972,11 +3069,58 @@ define([
                     castShadows : castShadows,
                     receiveShadows : receiveShadows,
                     uniformMap : uniformMap,
-                    renderState : rs,
+                    renderState : drawRS,
                     owner : owner,
                     pass : isTranslucent ? Pass.TRANSLUCENT : Pass.OPAQUE
                 });
 
+                // Setup the stencil command for the highlight
+                var highlightRS = clone(rs);
```

Comment:
Good catch, fixed and updated the Sandcastle demo to have an opacity slider, it looks nice with a partially transparent outline.